lib/grandpa/message_tracker.go

```diff
@@ -19,25 +19,26 @@ type tracker struct {
 	blockState BlockState
 	handler    *MessageHandler
 	votes      votesTracker
-
-	// map of commit block hash to commit message
-	commitMessages map[common.Hash]*CommitMessage
-	mapLock        sync.Mutex
-	in             chan *types.Block // receive imported block from BlockState
-	stopped        chan struct{}
+	commits    commitsTracker
+	mapLock    sync.Mutex
+	in         chan *types.Block // receive imported block from BlockState
+	stopped    chan struct{}
```

Comment:
```suggestion
	stopCh    chan struct{}
```

Reply:
Out of scope, changes show up since formatting (alignment) changed